### PR TITLE
Build arm64 for pharmer

### PR DESCRIPTION
**Pull request by lubinsz**
Signed-off-by: Bin Lu <bin.lu@arm.com>

What this PR does / why we need it:
add arm64/arm support

Which issue(s) this PR related:
#281 
    - pharmer: Build arm64 binary

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+all: pharmer 
+
+build_pharmer:
+	$(DOCKER_BUILD) 'CGO_ENABLED=0 $(SYSTEM) go build -o $(BINARY) $(VERBOSE_FLAG) $(GOTARGET)'
+
+pharmer:
+	for arch in $(LINUX_ARCH); do \
+		mkdir -p build/linux/$$arch; \
+		echo Building: linux/$$arch; \
+		$(MAKE) build_pharmer SYSTEM="GOOS=linux GOARCH=$$arch" BINARY="build/linux/$$arch/sonobuoy"; \
```
Comment (tamalsaha) [inline Makefile]: Why sonobuoy ?

Comment (lubinsz) [reply Makefile]: Sorry.
I have updated it. Please check it again.
Thanks.

**Comment by lubinsz** [comment]
@tamalsaha 
Can you take a look?

**Comment by lubinszARM** [comment]
ping @tamalsaha

**Comment by tamalsaha** [comment]
We have added ARM64 support in the new Makefile. https://github.com/pharmer/pharmer/commit/08517e7e7ff2e9935df09e9aa1aadf3464438317

Thanks for your contribution and patience, @lubinsz !

**Pull request closed without merging**